src/Steps/Step.php

```diff
@@ -109,15 +102,14 @@ public function runCommand(OutputInterface $output, $command, $error = null, $ex
 
         // Prepare unbound command
         if (is_array($command)) {
-            $process = ProcessBuilder::create($command)->getProcess();
+            $process = new Process($command);
         } elseif ($command instanceof Process) {
             $process = $command;
         } else {
-            $process = new Process($command);
+            $process = new Process([$command]);
         }
 
         // Set all default env vars
-        $process->inheritEnvironmentVariables();
```

Comment:
`[Symfony 5] The "Symfony\Component\Process\Process::inheritEnvironmentVariables()" method is deprecated since Symfony 4.4, env variables are always inherited`

Removed and no need for a replacement